### PR TITLE
Feature - Standings Offline update

### PR DESCRIPTION
**Pull request by ksookdhew**
Updated the standings offline functionality to use the Network monitor

### CODE REVIEW

**Review by sjacobs1 — APPROVED** [review]

**Review by kiasha10 — APPROVED** [review]

File: F1/F1/Standings/StandigsViewModel.swift [hunk 1]
```diff
@@ -17,8 +17,8 @@ class StandingsViewModel {
     let constructorViewModel: ConstructorStandingsViewModel
 
     init(navigationDelegate: StandingsNavigationDelegate?, delegate: ViewModelDelegate) {
-        self.driverViewModel = DriverStandingsViewModel(repository: DriverStandingsRepository())
-        self.constructorViewModel = ConstructorStandingsViewModel(repository: ConstructorStandingsRepository())
+        self.driverViewModel = DriverStandingsViewModel(repository: DriverStandingsRepository(coreDataManager: CoreDataManager()))
+        self.constructorViewModel = ConstructorStandingsViewModel(repository: ConstructorStandingsRepository(coreDataManager: CoreDataManager()))
         self.navigationDelegate = navigationDelegate
         self.delegate = delegate
     }
```
Comment (giacatano) [inline F1/F1/Standings/StandigsViewModel.swift:24]: Just check if there should be line spacing here

File: F1/F1/Standings/StandigsViewModel.swift [hunk 2]
```diff
@@ -73,4 +73,36 @@ class StandingsViewModel {
...
+
+        dispatchGroup.notify(queue: .main) { [weak self] in
+            if self?.driverStanding != nil && self?.constructorStanding != nil {
+                self?.delegate?.reloadView()
+            } else {
+                self?.delegate?.show(error: "Error")
+            }
+        }
+    }
+
```
Comment (giacatano) [inline F1/F1/Standings/StandigsViewModel.swift:107]: I think there might be an unnecessary line space here

File: F1/F1/Standings/StandigsViewModel.swift [hunk 3]
```diff
@@ -73,4 +73,36 @@ class StandingsViewModel {
...
 }
```
Comment (giacatano) [inline F1/F1/Standings/StandigsViewModel.swift:108]: There is a typo for standings on this file name

**Review by giacatano — COMMENTED** [review]
The code looks good, well done.

**Pull request merged**